### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for edit in "Create Authentication Files dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1387]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to edit and observe the name property using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
Tab
Destination folder:, read-only, C:\Users\abby, 
Shift
Tab
read-only, C:\Users\abby, 

**NVDA behaviour:**
Tab
Destination folder grouping
edit read-only, C:\Users\abby, 
Shift
Tab
edit read-only, C:\Users\abby, 

**Expected Result:**
Name property for the edit should be "Destination Folder".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create authentication file-edit-no name](https://user-images.githubusercontent.com/22956130/35682056-6b7e926c-0785-11e8-9d25-a4552a0ec3d5.jpg)
![create authentication file-edit-no name-nvda](https://user-images.githubusercontent.com/22956130/35682060-6e851580-0785-11e8-847c-295f6d804270.jpg)

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1387-fixed](https://user-images.githubusercontent.com/22956130/39762066-59a5921c-52f7-11e8-9aba-84d5815eaf3b.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749850
